test.py

```diff
@@ -62,15 +62,15 @@ def get_file_status(db_user, db_name, db_pass, db_host, file_id):
 
 def file2dataset_map(db_user, db_name, db_pass, db_host, file_id, dataset_id):
     """Assign file to dataset for dataset driven permissions."""
-    conn = psycopg2.connect(user=db_user, password=db_pass,
+    conn = psycopg2.connect(user='lega_out', password=db_pass,
```

Comment:
```suggestion
    conn = psycopg2.connect(user=db_user, password=db_pass,
```
on line 366 you already set it to that user, or did I get it wrong?

Reply:
missed to change the user